doc/Config_Editing_Guide.md

```diff
@@ -0,0 +1,166 @@
+	{
+		model="34thPRC_ArmourStandard_EOD_Helmet_Urban"; //This must be the same as the class name of the object it is a "variant" off. Basicly because we want all specialist helmets to be in the same toggle groups, we use the first object created in the group as an identifier for our AAX support.
+		helmet="commando"; //This can be referenced within 34thPRC_AceArsenalExtended, but essentially is used to mark which helmet this is for the toggles
+		camo="urban"; //Like the line above, but this identifies which camo version it is.
+	};
+```
+### Finishing The Model Swap
+```c++
+	// The below line is a little weird, essentially we are inheriting from MA_Armor's class ItemInfo which inherits from a built in class in arma. In this case it's HeadgearItem. This is basically a class used to define an item's properties. uniformModel links an object to a uniform. For more information see: https://community.bistudio.com/wiki/Arma_3:_Characters_And_Gear_Encoding_Guide
+	class ItemInfo : ItemInfo
+	{
+		uniformModel="MA_Armor\data\Helmets\Commando\Commando_Helmet.p3d"; //Here again we set this to the .p3d of our helmet
+	};
+};
+```
+### The Finished Example
+```c++
+class 34thPRC_ArmourStandard_Commando_Helmet_Urban : 34thPRC_ArmourStandard_Marines2_CH252_Urban
+{
+	displayName="[34th] Commando Helmet Urban";
+	author="Yandere";
+	model="MA_Armor\data\Helmets\Commando\Commando_Helmet.p3d";
+	hiddenSelections[]=
+	{
+		"Camo1",
+		"Camo2"
+	};
+	hiddenSelectionsTextures[]=
+	{
+		"34thPRC_ArmourStandard\data\marines2\specialist_helmets\commando\commando_helmet_urban.paa",
+		"34thPRC_ArmourStandard\data\mkvb\Commando\Visors\34_Commando_Helmet_Visor_Gold.paa"
+	};
+	class XtdGearInfo
+	{
+		model="34thPRC_ArmourStandard_EOD_Helmet_Urban";
+		helmet="commando";
+		camo="urban";
+	};
+	class ItemInfo : ItemInfo
+	{
+		uniformModel="MA_Armor\data\Helmets\Commando\Commando_Helmet.p3d";
+	};
+};
+```
+
+### Making Camo Variants
+```c++
+/* As a note, when you make other camo variants of a helmet, you need to inherit from the urban version which will allow you to skip a lot of the definitions and do something like the below:*/
+class 34thPRC_ArmourStandard_Commando_Helmet_Arid : 34thPRC_ArmourStandard_Commando_Helmet_Urban
+{
+	displayName="[34th] Commando Helmet Arid";
+	author="Yandere";
+	hiddenSelectionsTextures[]=
+	{
+		"34thPRC_ArmourStandard\data\marines2\specialist_helmets\commando\commando_helmet_arid.paa",
+		"34thPRC_ArmourStandard\data\mkvb\Commando\Visors\34_Commando_Helmet_Visor_Gold.paa"
+	};
+	class XtdGearInfo
+	{
+		model="34thPRC_ArmourStandard_EOD_Helmet_Urban";
+		helmet="commando";
+		camo="arid";
+	};
+};
+//This is not only cleaner but in the event that something is edited upstream or in a mod we inherit from we only need to change the base class
+```
+
+# Config Editing Tutorial 2

```

Comment:
I might put this section into a new file. Maybe make a folder in `docs` for all of the config editing guides.